webview/src/plots/components/ZoomedInPlot.tsx

```diff
@@ -91,7 +106,7 @@ export const ZoomedInPlot: React.FC<ZoomedInPlotProps> = ({
     actions: {
       compiled: false,
       editor: false,
-      export: true,
+      export: false,
```

Comment:
[F] As per [this thread](https://iterativeai.slack.com/archives/C01APS0FHDM/p1697438408439829) these actions don't work. We might be able to drop this altogether.

Reply:
need to keep the attribute or the actions do not show up